databases/week2/Part2.sql

```diff
@@ -0,0 +1,36 @@
+--Create a new database containing the following tables:
+create database homework;
+USE homework;
+create table `class`
+(
+`class_id` int not null auto_increment,
+`name` varchar
+(50) not null,
```

Comment:
line 8 append to line 7

Reply:
Actually, there is some problem with my VS Code , If I copy the statement from workbench it actually shows error and changes the format on save . 
I dont know what to do?